teampost.md

```diff
@@ -0,0 +1,28 @@
+Are you _tired_ of sifting through notifications? Clicking through so many browser tabs, getting notified so often it makes your computer's memory explode?? There's got to be a better way!
```

Comment:
Lol. Definitely needs a frustated person in black and white GIF :angry: :cry: :scream: